docs/app/Examples/elements/List/Types/ListDivided.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { List } from 'stardust'
+
+const ListDivided = () => {
+  return (
+    <List divided relaxed>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
+        <List.Content>
+          <List.Header as='a'>Semantic-Org/Semantic-UI</List.Header>
+          <List.Description as='a'>Updated 10 mins ago</List.Description>
+        </List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
+        <List.Content>
+          <List.Header as='a'>Semantic-Org/Semantic-UI-Docs</List.Header>
+          <List.Description as='a'>Updated 22 mins ago</List.Description>
+        </List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
```

Comment:
For clarity, we're talking about this:

![image](https://cloud.githubusercontent.com/assets/5067638/18848315/9c8cdb52-83e2-11e6-8d36-4649b16779e1.png)

Oh boy :/  The proper extrapolation of our API is to add `List.Icon`.  However, odds of users using this over `Icon` are slim.  Especially considering all other components use the `Icon` directly.

I think we add `verticalAlign` to the `Icon` itself.  I realize that prop would only be valid when used in a List, but it seems to make the most sense for users.

Any other wise ideas @jcarbo?